openid-4-verifiable-presentations-1_0.md

```diff
@@ -542,6 +545,266 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+# Verifiable Presentation Query Language {#vp_query}
```

Comment:
I would like to see this as a separate specification. It can be fully tested outside of OID4VP flows.

Reply:
i don't think the intention of this new query language has been to be protocol independent, so not sure what would be the benefit of "fully testing outside of OID4VP flows" unless I am misinterpreting it..

Reply:
My thought is that a software implementation of a query language is distinct from an implementation for OID4VP. It could have its own conformance suite and open source tooling, and potentially, utility outside of this specification.

If those broader ambitions are shared amongst the group then breaking it out could be the right path.